app/helpers/modLog.ts

```diff
@@ -139,7 +126,9 @@ const constructLog = ({
 ${reportedMessage}
 
 ${postfix}`,
-        embeds: [{ description: `${link}${attachments}` }],
+        embeds: [
+          { description: `${link}${attachments ? `\n\n${attachments}` : ""}` },
```

Comment:
I guess lines 130, 147, 158 are using the same object (regarding it's contents). Hence IMO it can be put in a single variable and reused. Maybe even whole embeds array?

Reply:
Fair point, but I think this whole section is getting complex enough to need a refactor. There are 2 different types of embeds, one with a link to the original message and one without. Since there are variations, atm I have a slight preference to duplicate